common/src/Microsoft.Azure.IIoT.Hub.Processor/src/Services/EventProcessorFactory.cs

```diff
@@ -308,7 +330,17 @@ private class EventProperties : IDictionary<string, string> {
             private static readonly Gauge kEventProcessorDetails = Metrics
                 .CreateGauge("iiot_event_processor_info", "details about event processor",
                     new GaugeConfiguration {
-                        LabelNames = new[] { "id", "partition_id", "status" }
+                        LabelNames = new[] { "id", "eventhub_name", "consumer_name", "partition_id", "status" }
+                    });
+            private static readonly Gauge kOldEvent = Metrics
+                .CreateGauge("iiot_event_processor_skipped_events", "number of messages skipped due to being old",
+                    new GaugeConfiguration {
+                        LabelNames = new[] { "id", "eventhub_name", "consumer_name", "partition_id" }
+                    });
+            private static readonly Gauge kEventProcessorMessages = Metrics
+                .CreateGauge("iiot_event_processor_message_count", "number of messages processed",
```

Comment:
Please use term `events` instead of `messages` in `iiot_event_processor_message_count`. Having both `events` and `messages` here and in `iiot_event_processor_skipped_events` implies that there is a difference.

Reply:
I agree.